deepchecks/nlp/text_data.py

```diff
@@ -89,13 +90,18 @@ class TextData:
     categorical_properties : t.Optional[t.List[str]] , default: None
         The names of the categorical properties columns. If None, categorical properties columns are automatically
         inferred. Only relevant if properties is not None.
+    embeddings : t.Optional[Union[pd.DataFrame, str]] , default: None
```

Comment:
I am against the 'auto' option, calculating is a heavy operation which can be buggy and as such should be executed after text data is created. Similar to properties calculation

Reply:
Doesn't really exist, it's a mistake in the docstring. Fixed in the next PR of docs

Reply:
Fixed now